packages/front-end/components/Metrics/DateGraph.tsx

```diff
@@ -277,54 +302,136 @@ const DateGraph: FC<{
     });
   }
 
+  const { containerRef, containerBounds } = useTooltipInPortal({
+    scroll: true,
+    detectBounds: true,
+  });
+
+  const width = (containerBounds?.width || 0) + marginRight + marginLeft;
+  const dateNums = data.map((d) => getValidDate(d.d).getTime());
+  const min = Math.min(...dateNums);
+  const max = Math.max(...dateNums);
+  const yMax = height - marginTop - marginBottom;
+  const xMax = containerBounds?.width || 0;
+  const numXTicks = width > 768 ? 7 : 4;
+  const numYTicks = 5;
+  const axisHeight = 30;
+  const minGraphHeight = 100;
+  const expBarHeight = 10;
+  const expBarMargin = 4;
+  const expHeight = bands.size * (expBarHeight + expBarMargin);
+  let graphHeight = yMax - expHeight;
+  if (graphHeight < minGraphHeight) {
+    height += minGraphHeight - (yMax - expHeight);
+    graphHeight = minGraphHeight;
+  }
+  const xScale = useMemo(
+    () =>
+      scaleTime({
+        domain: [min, max],
+        range: [0, xMax],
+        round: true,
+      }),
+    [min, max, xMax]
+  );
+
+  const yScale = useMemo(
+    () =>
+      scaleLinear<number>({
+        domain: [
+          0,
+          Math.max(...data.map((d) => Math.min(d.v * 2, d.v + (d.s ?? 0) * 2))),
```

Comment:
Do we need to support binomial here again?

Reply:
Not specifically no. `d.v` (datapoint's value) will capture what we need for a binomial variable.